walkthroughs/3-low-code-api-development/walkthrough.adoc

```diff
@@ -35,12 +33,15 @@ It also allows you to identify important design decisions prior to coding and de
 .Red Hat OpenShift
 ****
 * link:{openshift-console-url}[Console, window="_blank"]
+* link:https://help.openshift.com/[OpenShift Online Help Center, window="_blank"]
+* link:https://blog.openshift.com/[OpenShift Blog, window="_blank"]
 ****
 
 [type=walkthroughResource,serviceName=fuse]
 .Red Hat Fuse Online
 ****
-* link:{fuse-url}[Console, window="_blank"]
+* link:{fuse-url}[Console, window="_blank", id="resources-fuse-url"]
+* link:{fuse-documentation-url}[Fuse Documentation, window="_blank"]
```

Comment:
This `fuse-documentation-url` doesn't appear to be defined. Again, I can address it later.

Reply:
Ok, it's possible I didn't carry over an attribute for this. The idea here is the same as for OpenShift resources. If a SP includes Fuse, it should be consistent with what the resources are and where they link to.